openhands/agenthub/codeact_agent/tools/browser.py

```diff
@@ -18,6 +18,11 @@
 fill('a12', 'example with "quotes"')
 click('a51')
 click('48', button='middle', modifiers=['Shift'])
+
+You can also use the browser to view pdf, png, jpg files.
+You should first check the content of /tmp/oh-server-url to get the server url, and then use it to view the file by `goto("{server_url}/view?path={absolute_file_path}")`.
```

Comment:
I have a stupid question: isn't `server_url` always "127.0.0.1"?

Reply:
yes, but the port could be very different across different environment (saas, docker), and we need to tell the agent the port 😢

Reply:
Ah yeah I forgot about that.

So in the long-term, do we want to make prompts templating languages, and then just fill in the port info during runtime?